recipe/meta.yaml

```diff
@@ -13,15 +13,17 @@ source:
 
 build:
     noarch: python
-    script: {{ PYTHON }} -m pip install . --no-deps -vv
-    number: 0
+    script:
+      - export SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+      - {{ python }} -m pip install . --no-deps
```

Comment:
Try putting this on a single line: 

```
SETUPTOOLS_SCM_PRETEND_VERSION={{ version }} {{ python }} -m pip install . --no-deps
```

Reply:
Tried that one [here](https://github.com/conda-forge/pint-feedstock/pull/34/commits/2381eaa7cc26b9ea5bdf89ff38b83424389e4679).

Reply:
:man_shrugging:

Reply:
Yea, I feel the same way. I think the short is we are going to recommend everyone to stay away from `setuptools_scm` for now.

Reply:
Yeah, I agree. It is pretty annoying to deal with as a downstream packager

Reply:
Needed `toml`... either way success!